src/js/jsx/shared/ColorPicker.jsx

```diff
@@ -403,18 +406,107 @@ define(function (require, exports, module) {
                     onMouseUp={this._handleMouseUp}
                     onMouseDown={this._handleMouseDown}
                     onTouchStart={this._startUpdates}>
+                    <div
+                        className="color-picker-map__background"
+                        style={{
```

Comment:
In your render function (above the return) define to additional variables:
```javascript
var backgroundStyle = {
        backgroundColor: this.props.backgroundColor
    },
    pointerStyle = {
        left: this._getPercentageValue(this.props.x),
        bottom: this._getPercentageValue(this.props.y)
    };
```

Then in your the JSX returned by the render function, just do:
```html
<div
    className="color-picker-map__pointer"
    style={pointerStyle} />
```
Etc.